px-progress-bar.html

```diff
@@ -74,28 +60,61 @@
         value: false,
         reflectToAttribute: true,
         observer: 'ready'
+      },
+      /**
+       * The minimum value for the progress bar, used to calculate the amount
+       * that is filled in.
+       */
+      min: {
+        type: Number,
+        value: 0
+      },
+      /**
+       * The maximum value for the progress bar, used to calculate the amount
+       * that is filled in.
+       */
+      max: {
+        type: Number,
+        value: 100
+      },
+      /**
+       * Internal property for calculating the ratio (from 0 to 1) for sizing
+       * elements within the component - also clips negative values and values
+       * greater than 100 in its compute function.
+       */
+      _ratio: {
+        type: Number,
+        computed: '_computeRatio(value, min, max)',
+        observer: '_updateProgress'
       }
     },
-     /**
+    attached: function() {
+      this.setAttribute('role', 'progressbar');
+      this.setAttribute('aria-valuemin', '0');
+      this.setAttribute('aria-valuemax', '100');
```

Comment:
don't know much about the aria stuff, but if you set it there shouldn't you be setting it on min/max change as well?

Reply:
yep good plan.